errata/static/errata/js/lib/bootstrap/css/bootstrap.min.css

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
Is this the same bootstrap.min.css references above?

Reply:
Yes, there was another copy in the javascript folder. It's been removed.